ulib/experimental/Steel.Effect.fst

```diff
@@ -109,25 +109,25 @@ let repr a pre post req ens =
 let returnc (a:Type u#a) (x:a) (p:a -> slprop) = fun _ -> x
 
 let bind a b pre_f post_f req_f ens_f post_g req_g ens_g f g
-  = fun m0 ->
-      let x = f () in
-      g x ()
+  = fun frame ->
+      let x = f frame in
+      g x frame
```

Comment:
Surprising diff ... I guess the original `m0` was poorly named, it's just a unit.